### DISCUSSION TITLE
An Application Error Occurred: client side error occurred.

### DISCUSSION THREAD

**Discussion by vgseven** [vercel/community#1015] [Help] [answered]
After Deployment of website at vercel or even at firebase thevgnote.web.app face an application error but site work properly at local host.

**Comment by amyegan** [comment, marked as answer]
Hi @vgseven! It looks like you're getting the error `Cannot read properties of null (reading 'content')`, so I would recommend adding a try/catch and [logging](https://vercel.com/docs/concepts/deployments/logs#function-logs) around anywhere that you are trying to access `content`. That will help you get more details about what is failing and why.

If you need more help, it's usually helpful for me to get a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so I can help with debugging. 🙏

**Reply by vgseven** [reply]
Solved, it's happens due invalid placing of some tags..
